maven_iuvs/download.py

```diff
@@ -131,11 +130,13 @@ def setup_user_paths():
     user_paths_file.write("iuvs_vm_username = \""+vm_username+"\"\n")
     user_paths_file.write("auto_spice_load = "+auto_spice_load+"\n")
     user_paths_file.close()
-    # now scripts can import the relevant directories from user_paths
 
+    # now scripts can import the relevant directories from user_paths
 
+ 
 def get_default_data_directory(level='l1b'):
-    """Returns default l1b_dir defined in user_paths.py, creating the
+    """
```

Comment:
Yes, the original code (perhaps written by you or Kyle or both) contained the summary lines starting on the same line as the opening triple quote (mostly, I think). In some of my functions I started the summary on the next line because that looks better to me without really thinking about it/checking what PEP actually says. In undoing some of the messy commits, I tried to move these back to their original location on the same line as """ to ensure line numbers would match so that the overall diff would be cleaner for you [and others] to read. 

I did just check PEP and it says that either is acceptable:

> The summary line may be on the same line as the opening quotes or on the next line.

I'm currently using SublimeLinter with flake8 conventions. We can discuss to agree on a convention and then I can make a new commit later that just focuses on cleaning up any formatting issues in the docstrings/other comments!

Reply:
Let's open an issue on this

Reply:
Issue https://github.com/lasp/maven_iuvs/issues/58 is opened, resolving this conversation